internal/controllers/hardwareplugin_manager_setup.go

```diff
@@ -197,6 +197,16 @@ func (t *reconcilerTask) createHardwarePluginManagerClusterRole(ctx context.Cont
 					"update",
 				},
 			},
+			// HardwarePlugin registration verification
+			{
+				NonResourceURLs: []string{
+					"/hardware-manager/provisioning/*",
+					utils.HardwarePluginValidationEndpoint,
```

Comment:
Is this line necessary if the wildcard above it already exists?

Reply:
Agreed!